libsolidity/ast/AST.h

```diff
@@ -435,6 +437,7 @@ class ContractDefinition: public Declaration, public Documented
 	std::vector<ASTPointer<InheritanceSpecifier>> m_baseContracts;
 	std::vector<ASTPointer<ASTNode>> m_subNodes;
 	ContractKind m_contractKind;
+	bool m_isAbstract{false};
```

Comment:
Needs a function that returns `m_isAbstract`. Also please just use `m_abstract`.